### PR TITLE
Add dots to represent that disc can be moved

### PR DESCRIPTION
**Pull request by Ojself**
[YES-6](https://app.gitkraken.com/glo/view/card/5c865d947ec843cca3a6cde9e1a580be)

- Added dots that expands on hover || clicked.
- Shows ghost discs with alpha/transparency when moveable disc is clicked
- removed hovering from player discs when phase is in mid_move
- Phase is now set for every move that is being made
- adjusted framerate from default and down to 20. This cause a eyesore when looking at the 'lagging' sun in Background
- Add state in Hud class to indicate what player should do next (for the first 3 moves)

### CODE REVIEW

File: lib/Game.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const PHASES = Object.freeze({
 class Game {
 	constructor({
 		pid = nanoid(),
-		players = ['Max', 'Tor'],
+		players = this.players || ['Max', 'Tor'],
```
Comment (Parnswir) [inline lib/Game.js:18]: what should this line do? `this.players` should always be undefined here. `['Max', 'Tor']` already is the default for the parameter. If it is set, the value is overridden.

File: public/scripts/main.js [hunk 2]
```diff
@@ -9,6 +11,7 @@ function preload() {
 
 function setup() {
 	createCanvas(1200, 1000);
+	frameRate(20)
```
Comment (Parnswir) [inline public/scripts/main.js:14]: y tho?

File: public/scripts/menu.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    draw() {
+        this.drawTitle()
+        if (this.showInstructions) {
+            this.drawInstructions()
+        }
+    }
+
+    drawTitle() {
+        text('NONAGA', 100, 100)
```
Comment (Parnswir) [inline public/scripts/menu.js:20]: I think you spelled "yesnaga" wrong :-)

**Review by Parnswir — CHANGES_REQUESTED** [review]

File: public/scripts/menu.js (hunk 3 above)
Comment (Ojself) [reply public/scripts/menu.js:20]: oops:D

File: public/scripts/main.js (hunk 2 above)
Comment (Ojself) [reply public/scripts/main.js:14]: It got a bit cpu intensive for my computer so I had to lower it. I will adjust it back

File: lib/Game.js (hunk 1 above)
Comment (Ojself) [reply lib/Game.js:18]: Aha. I thought I needed it if the player didn't provide any `players` to the `.put`request. 
Will adjust it now

**Review by Parnswir — APPROVED** [review]

**Pull request merged**